import-secrets/Gopkg.toml

```diff
@@ -0,0 +1,193 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "github.com/bitnami-labs/sealed-secrets"
+  version = "0.7.0"
+
+[[constraint]]
+  name = "gopkg.in/yaml.v2"
+  version = "2.2.1"
+
+[[constraint]]
+  branch = "master"
+  name = "k8s.io/api"
+
+[[constraint]]
+  branch = "master"
+  name = "k8s.io/apimachinery"
+
+[prune]
+  go-tests = true
+  unused-packages = true
+
+# everything below need to be present for k8s.io/client-go to work with dep
+# https://github.com/kubernetes/client-go/blob/master/INSTALL.md#dep-not-supported-yet
+
+[[constraint]]
+  name = "k8s.io/client-go"
+  version = "7.0.0"
+
+[[override]]
+  name = "cloud.google.com/go"
+  revision = "3b1ae45394a234c385be014e9a488f2bb6eef821"
+
+[[override]]
+  name = "github.com/Azure/go-autorest"
+  revision = "1ff28809256a84bb6966640ff3d0371af82ccba4"
+
+[[override]]
```

Comment:
This is way more packages than I expected including jwt-go... how come they are needed here but not in our samples?

Reply:
This is done because of **[client-go]**(https://github.com/kubernetes/client-go/blob/master/INSTALL.md#dep-not-supported-yet): 
> As a workaround, which may or may not be worthwhile, you can specify all client-go dependencies manually as override in Gopkg.toml with the versions listed in Godeps.json, and manually update them when you upgrade client-go version.